### ISSUE TITLE
Improve way of changing surface group for surface

### ISSUE THREAD

**Issue by PatrykSzwer** [Starcounter/Blending#11]
_Issue moved from Website repository: https://github.com/Starcounter/Website/issues/149_

Right now group in specific surface can be changed by editing it in general settings of the surface. It's working solution, but not as intuitive as It could be. I suggest to add drag and drop functionality (probably with juicy-draggable) in Surface groups view, to make possible dragging surface from one group and dropping it in another.

**Comment by warpech** [comment]
I think that this happens rarely so it does not need any improvement. Should we keep it in the backlog or close until there is user evidence that this is needed? I'd vote for closing.

**Comment by PatrykSzwer** [comment]
I agree that this can happen rarely, but on the other hand I think It's inconsistency in our interface (because you have to leave groups view to change something about groups) that is conspicuous and sooner or later should be fixed.
